nms/app/context/CbsdContext.tsx

```diff
@@ -69,10 +62,13 @@ async function fetch(params: {
       })
     ).data;
     setFetchResponse(response);
-  } catch {
-    enqueueSnackbar?.('failed fetching CBSDs information', {
-      variant: 'error',
-    });
+  } catch (error) {
+    console.error(error);
+    // #469 Ignore error when domain proxy module is disabled,
```

Comment:
Maybe link to https://github.com/magma/domain-proxy/issues/469 instead because it is not clear that this isn't an issue from the main repo.

Reply:
Makes sense, corrected :)